src/main/java/io/jenkins/plugins/synopsys/security/scan/extension/freestyle/SecurityScanFreestyle.java

```diff
@@ -733,6 +835,76 @@ public void setMark_build_status(String mark_build_status) {
         this.mark_build_status = mark_build_status;
     }
 
+    @DataBoundSetter
+    public void setSrm_url(String srm_url) {
+        this.srm_url = srm_url;
```

Comment:
Use `Util.fixEmptyAndTrim()`

Reply:
I saw many fields missing fixEmptyAndTrim that are unavailable in the pipeline syntax. So it's fine.